### ISSUE TITLE
Unable to clear:cache for test environment

### ISSUE THREAD

**Issue by covex-nn** [symfony/symfony#25189]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.4-RC2, 4.0-RC2

When i run `php bin\console cache:clear --env=test` i get a error message:
```
C:\sf\4>php bin\console cache:clear --env=test

In Client.php line 28:

  Class Symfony\Component\HttpKernel\Client not found


In Client.php line 31:

  Class Symfony\Component\BrowserKit\Client not found
```

There is no error messages for `dev` and `prod` environments.

To reproduce error download `composer.json` from `symfony/skeleton`, run `composer install` and `php bin\console cache:clear --env=test`:

* https://github.com/symfony/skeleton/blob/3.4/composer.json
* https://github.com/symfony/skeleton/blob/4.0/composer.json

There is no errors with `composer.json` from `symfony/skeleton@3.3`

**Comment by aar0nr** [comment]
This is because the Framework bundle is [registering](https://github.com/symfony/framework-bundle/blob/v4.0.0-RC2/Resources/config/test.xml#L14) the ```test.client``` service even if the ```Symfony\Component\BrowserKit\Client``` is not available.

To fix, run: ```composer require symfony/browser-kit --dev```

**Comment by covex-nn** [comment]
@aar0nr `test.client` service existed in all previous versions too, but there was no error messages. So, i think a bug can be somewhere in updated ContainerBuilder since v3.4: file `vendor/symfony/framework-bundle/Client.php` is always required even if it not used in application.

**Comment by covex-nn** [comment]
Already fixed in #25160
